distributionviewer/core/static/css/distributionviewer.styl

```diff
@@ -22,6 +22,11 @@ a {
 .global-wrapper {
   margin: 0 auto;
   max-width: 1400px;
+  padding: 10px;
+
+  .filtered & {
+    padding: 10px 0 0 $filters-width;
```

Comment:
It does for now since '.filters' is 'absolute'. I want to make everything flexed for the next iteration but felt that PR would be too massive to include here also.